prow/cmd/deck/static/script.js

```diff
@@ -549,30 +549,37 @@ function createRerunCell(modal, rerun_command, prowjob) {
 
 function stateCell(state) {
     const c = document.createElement("td");
-    c.className = state;
+    c.classList.add(...["icon-cell"]);
```

Comment:
dont need to unwrap the array
`c.classList.add("icon-cell");`

Btw, I dont see where is `.icon-cell`'s usage

Reply:
icon-cell is just in line with the other td's with icons here.

Reply:
fixed the ...